cmake/ElektraCache.cmake

```diff
@@ -365,14 +365,8 @@ set (LIB_SUFFIX ""
 		"Optional suffix to use on lib folders (e.g. 64 for lib64)"
     )
 
-set (MEMORYCHECK_COMMAND
```

Comment:
> are you sure this won't break make memcheck. Any reason why you removed it?

I am not sure about other operating systems, but on my machine `make memcheck` would not work if this values was set. That was even the case, when I set the path correctly for my setup (`/usr/local/bin/valgrind`). The command would always complain that it could not locate `MemoryCheckCommand` or that the specified path for `MemoryCheckCommand` was wrong.